pwem/__init__.py

```diff
@@ -83,5 +83,4 @@ def defineBinariesMaxit(cls, default, env):
         # requirements bison, flex, gcc
 
 
-Domain.registerPlugin(__name__)
```

Comment:
why this is gone? all pwem protocols fail now!

Reply:
Need an entry point and be installed again